tapir/shifts/tests/test_shift_template_duplicate.py

```diff
@@ -0,0 +1,50 @@
+from django.urls import reverse
+import datetime
+
+from tapir.utils.tests_utils import TapirFactoryTestBase
+from tapir.shifts.tests.factories import ShiftTemplateFactory
+from tapir.shifts.models import ShiftTemplate, ShiftTemplateGroup
+
+
+class TestShiftTemplateDuplicate(TapirFactoryTestBase):
```

Comment:
We need tests to be more explicit. For example:

- one that checks that the copy is correct: do one single copy and check that the fields are copied correctly.
- one that checks that we don't re-create the same shift template: send the same day and group as post parameters and check that no shift is created.